dhcpv4/ztpv4/ztp_test.go

```diff
@@ -70,3 +71,48 @@ func TestParseV4VendorClass(t *testing.T) {
 		})
 	}
 }
+
+func TestParseV4VIVC(t *testing.T) {
```

Comment:
Can you remove the V4 from the test name? I think it is redundant.

I know the previous test had it.